### DISCUSSION TITLE
Any way to wire the ESP32 buttons to somewhere else?

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#70] [Q&A]
Is there any way to use any of the pins on the ESP32 to 'replicate' the button functionality that's on board?
Working on an internal wire on the gamecube and would be nice to have them accessible

Thanks!

**Comment by Zonark** [comment]
If your talking boot button I would just attach wires to the bottom of the button then add another button wherever you want to move it to

that will bypass the button and still press it basically
    |b|
    |   |
+_|b|_-

kinda like that

**Reply by Zonark** [reply]
I have now successfully implemented this and now can confirm it works 100%
what I did is add solder to the pads on the boot button, added wires, then attached to another button I had moved to the external side of my case.
